org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/DeviceRequestTest.java

```diff
@@ -89,50 +102,55 @@ public void createServices() throws Exception {
 		info.setIcon("org.eclipse.scanning.example/icon/mandelbrot.png");
 		mandy.setDeviceInformation(info);
 		((RunnableDeviceServiceImpl)dservice)._register("mandelbrot", mandy);
-		
+
+		final DummyMalcolmDevice malc = new DummyMalcolmDevice();
+		final DeviceInformation<DummyMalcolmModel> malcInfo = new DeviceInformation<>();
+		malcInfo.setName("malcolm");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "malcolm" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)